### PR TITLE
Update Vulnerability-Scan.md

### PR DESCRIPTION
**Pull request by maanasagovi**
removed images with sensitive information

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: getting-started/Vulnerability-Scan.md [hunk 1]
```diff
@@ -36,20 +36,14 @@ Before beginning the pre-scan process, you will need to set up a Microsoft Azure
 
 6. You can select each description to view severity categories and begin remediating (you will also be able to see manual remediation steps as well) 
 
-<p align="center"><img src="images/step6-vulnerabilities.png" alt="step6-vulnerabilities" width="800"/>
```
Comment (thomasyip-msft) [inline getting-started/Vulnerability-Scan.md:39]: remove the actual image too?

Comment (janeguo-msft) [reply getting-started/Vulnerability-Scan.md:39]: Yeah, I think you should.

Comment (janezi7guo) [reply getting-started/Vulnerability-Scan.md:39]: > remove the actual image too?

Yeah, I think you should.

**Review by janezi7guo — CHANGES_REQUESTED** [review]
Please remove the image files in the folder as well.

**Review by janezi7guo — APPROVED** [review]

**Comment by janezi7guo** [comment]
#sign-off

**Pull request merged**